src/bootstrap-table.js

```diff
@@ -831,6 +832,7 @@
         styles: [],
         classes: [],
         formatters: [],
+        detailFormatters: [],
```

Comment:
We need to `detailFormatter` to `COLUMN_DEFAULTS` like other column options.

Reply:
done